dco.txt

```diff
@@ -0,0 +1,37 @@
+Developer Certificate of Origin
```

Comment:
Out of curiosity, I have seen repositories where, in order to contribute, a CLA had to be signed.  Is this DCO an equivalent to a CLA by an individual?

Reply:
Oh I forgot to commit the CLA, sorry about that. I'm doing it now.

CLA and DCO are complementary: CLA defines the terms of the IP for contributors on the project (starting from his first contribution in the project), DCO is more for code donation that has been made outside of the project.

The good point is that both can be omitted when using the Apache License (as the License already describes the terms).
However, I like to provide both for clarity for contributors.

Up to you here: we can remove DCO, or I can add CLA 😄

Reply:
The ASL only says that all Contributions are licensed under the ASL.  Although it is valid, I think it only _implicitly_ says that one must be the author (or authorized by the author) to submit such Contributions.  I like the fact that the DCO makes these assumptions explicit.  I would rather keep it.

AFAIU a CLA will only add that all contributions are (c) Dremio 2024.  Is that correct?  I don't mind adding it, especially if you want this project to be the guinea pig for future OSS projects at Dremio.

Reply:
Yes that's my point. Even if it would not be strictly required in Peco case, it's a good example and validation of the process I'm writing 😃

Reply:
Ok, +1 to add the CLA then